src/font/text.cpp

```diff
@@ -204,7 +230,13 @@ point pango_text::get_cursor_position(const unsigned column, const unsigned line
 
 	// Go the wanted line.
 	if(line != 0) {
-		if(pango_layout_get_line_count(layout_.get()) >= static_cast<int>(line)) {
+// 		Doesn't seem right -- babaissarkar
+//
+//		if(pango_layout_get_line_count(layout_.get()) >= static_cast<int>(line)) {
+//			return point(0, 0);
+//		}
+
+		if(static_cast<int>(line) >= pango_layout_get_line_count(layout_.get())) {
```

Comment:
So basically, you changed the comparison from "greater or equal" to "less than", right? Is there a reason not to just use "less than"? It would just make the diff a little clearer (as would deleting the comments).

Reply:
Cleaned up.

Reply:
> So basically, you changed the comparison from "greater or equal" to "less than", right?

Changed "greater/equal" to "less/equal", actually.